src/components/dashboard/FaceVerification/screens/VerificationScreen.js

```diff
@@ -53,21 +52,10 @@ const FaceVerification = ({ screenProps }) => {
 
   const retryHandler = useCallback(
     eventData => {
-      const exception = eventData.reason
-
-      // tracking all liveness exceptions except
-      // the last one required to show 'try again lagter'
-      if (attemptsCount < MAX_RETRIES_ALLOWED) {
-        trackAttempt(exception)
-      } else if (!livenessExceptionRef.current) {
-        // storing the last one in the ref, it will be
-        // tracked & reported by the error screen
-        livenessExceptionRef.current = exception
-      }
-
+      trackAttempt(eventData.reason)
```

Comment:
@serdiukov-o-nordwhale what value eventData.reason contain? is it exception? if not it could fail inside trackAttempt.

Reply:
yes it's an exception catched in enrollmentprocessor.web.js